tst/EnergyPlus/unit/KusudaAchenbachGroundTemperatureModel.unit.cc

```diff
@@ -58,11 +58,10 @@
 
 using namespace EnergyPlus;
 using namespace EnergyPlus::GroundTemperatureManager;
- 
+
 TEST_F( EnergyPlusFixture, KusudaAchenbachGroundTempModelTest1 )
 {
 	std::string const idf_objects = delimited_string({
-		"Version,8.4;",
```

Comment:
Why delete these version statements?  Don't they help date the vintage of the idf snippet

Reply:
Don't the IDF objects in the unit tests have to undergo transition along with all of the IDF files to stay current with the code? To my mind, it doesn't seem necessary to keep the version number of the original unit test around even as the IDF snippets stay up to date with current code. Maybe we start adding a standard header to each test stating the author and version it when it was created? Or just rely on git blame to show the history. I don't have a horse in this race, so whatever is decided is fine with me.

Reply:
The version numbers in the unit test snippet will almost always mean: "the oldest version of E+ that these inputs were valid, or the version the unit test was added."  As soon as the unit tests fail because the object changes, I'm sure the developer will probably update the version number.  But to what gain?  I don't really see the value of the version number in those.  As long as we are running the unit tests on each commit, we'll know immediately when one fails due to bad inputs so it should be quite obvious what needs to be updated to make it work again.

Reply:
While I agree in principal, just having a failing unit test could be from any number of issues. Could be bad code, unit test code that needs to change, IDF snippet version needs to be updated, etc. In my opinion, having the Version number in the snippet provides a quick visual reference of what version the IDF snippet is last guaranteed to work. This can provide an easy reference to debug a failing unit test when it is the snippet that needs to be version translated. That said, as long as `process_idf()` is wrapped with `ASSERT_FALSE` then that should be enough to easily and quickly catch IDF snippet input issues.